app/views/application/_dashboardstats.html.erb

```diff
@@ -1,8 +1,14 @@
 <section class='dashboard-stats'>
 
+  <section class='yearly-revenue'>
+    <h3>Yearly Revenue</h3>
+    <%= bar_chart @yearly_revenue.group_by_month(:created_at, format: "%b %Y").count %>
```

Comment:
ack, this is an awful lot of data crunching to be done in the view, and would be better to be pulled at the controller level, and then passed as something like `<%= bar_chart @pre_processed_data %>`